variables.tf

```diff
@@ -83,19 +83,19 @@ variable "application_type" {
 
 variable "daily_data_cap_in_gb" {
   description = "(Optional) Specifies the Application Insights component daily data volume cap in GB."
-  default     = null
+  default     = 0.15
   type        = number
 }
 
 variable "daily_data_cap_notifications_disabled" {
   description = "(Optional) Specifies if a notification email will be send when the daily data volume cap is met."
-  default     = null
+  default     = false
```

Comment:
Default Tensio@Computas.no?

Reply:
Denne [daily_data_cap_notifications_disabled](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_insights#daily_data_cap_notifications_disabled) argument er boolean typen. Så den default verdien er riktig.

Finnes ikke argumenten til å sette varsling epost fra [terraform_application_insights](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_insights) provider.

Skal undersøke om hvordan setter oss epost gjennom Terraform for denne.